Product plan/Product-plan.tex

```diff
@@ -83,62 +85,93 @@ \subsubsection*{Could haves}
 	\item It could be possible for an agent to reason about other agents emotional bases.
 \end{itemize}
 
-\subsubsection*{Won't haves}
+\subsubsection*{Will not haves}
 \begin{itemize}
-	\item eclipse plug-in
+	\item Eclipse plug-in
 \end{itemize}
 
 \subsection{Road map}
-\begin{itemize}
-	\item week 1: seminars
-	\item week 2: understanding Gamygdala by testgame
-	\item week 3: setting up goal and research on both goal and Gamygdala.
-	\item week 4: agents in goal should get in Gamygdala their equivalent + goals. relations between agent/goals definition/
-	\item week 5:  Extra mental database for emotions of an agent. Retrieve emotions.
-	\item week 6: debug tool + updating relations (between believes/goals agents/agents) + setting gain/decay function
-	\item week 7: implementing ability to set custom decay function
-	\item week 8: used for tasks that are not finished
-	\item week 9: used for tasks that are not finished
-	\item week 10: presentations + finishing
-	
-\end{itemize}
+In this section we will list for each week what we will be doing. At the end of each week the user stories that will be completed in that week are listed.
+\subsubsection*{Week 1}
+The first week we mainly have seminars about the project and how things will go. In this week we will setup the group and make sure everybody gets ready to work on the project. Setting up communication between group members for example.
+
+\subsubsection*{Week 2}
+During this week we will implement 2 very small and easy games in Javascript. The games will work with Gamygdala. We create this games so that each member can get a better understanding of Gamygdala and how it reasons.
+
+\subsubsection*{Week 3}
+This week the focus will be on setting up the repositories. Furthermore there will be a lot of research on where and how to integrate Gamygdala. Also the first steps towards the product will be implemented. The Gamygdala port will be instantiated in GOAL and a parser for the emotional configuration will be completed. This research is handy so that we do not implement wherever we can but really know what to do and where.(1,2)
```

Comment:
Misschien beter iets in de zin van "This research is useful so that we implement the functions of Gamygdala in the correct places to maintain good code quality."